apps/modernization-ui/src/apps/patient/profile/PatientProfile.tsx

```diff
@@ -18,11 +17,7 @@ import { TabNavigationEntry, TabNavigation } from 'components/TabNavigation/TabN
 
 const openPrintableView = (patient: string | undefined) => () => {
     if (patient) {
-        window.open(
-            `${Config.nbsUrl}/LoadViewFile1.do?method=ViewFile&ContextAction=print&uid=${patient}`,
-            '_blank',
-            'noreferrer'
-        );
+        window.open(`/nbs/LoadViewFile1.do?method=ViewFile&ContextAction=print&uid=${patient}`, '_blank', 'noreferrer');
```

Comment:
didn't want to leave nbsUrl configurable?

Reply:
The existing configuration adds the protocol and host to make a fully qualified path.  The NBS 6 and NBS 7 UIs should always be on the same domain allowing us to use relative URLs.